GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -885,7 +885,7 @@ private void ProcessApplyOutput(string output, byte[] patch)
         {
             if (!string.IsNullOrEmpty(output))
             {
-                MessageBox.Show(this, output + "\n\n" + SelectedDiff.Encoding.GetString(patch));
+                MessageBox.Show(this, output + "\n\n" + SelectedDiff.Encoding.GetString(patch), "Information", MessageBoxButtons.OK, MessageBoxIcon.Information);
```

Comment:
if `output` is not empty, it means there has been warnings/errors during the execution of the git command, so it arguably this must be an error.